pkg/testcase/cluster.go

```diff
@@ -4,12 +4,12 @@ import (
 	"fmt"
 	"strings"
 
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
```

Comment:
lets add this on the last imports
order:

- internal go
- internal pkg
- internal project
- external libs

Reply:
fixed